screens/LoginScreen.js

```diff
@@ -375,91 +377,95 @@ class LoginScreen extends React.Component {
     const passwordErrorMessage = passwordValidation ? <Text style={styles.validationErrorMessage}>{i18n.t('login.password.error')}</Text> : null;
 
     return (
-      <View style={styles.container}>
-        <View style={styles.header}>
-          <Image
-            source={require('../assets/images/dt-logo2.png')}
-            style={styles.welcomeImage}
-          />
-        </View>
+      <KeyboardAvoidingView behavior="padding">
+        <ScrollView>
+          <View style={styles.container}>
```

Comment:
Can you combine the `ScrollView` and the child `View` with the container styles on it so we don't have two wrapper components? I think it would become `<ScrollView contentContainerStyle={styles.container}>`